frontend/src/app/login/login.drv.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Directive for login formular
+ */
+angular.module('flocs.user')
+.directive('loginForm', function(){
+	return{
+		restrict:'EA',
+		template:
```

Comment:
For file templates, use the templateUrl attribute instead of template attribute: templateUrl: 'componentDirectory/templateName.tpl.html', for example templateUrl: 'home/home.tpl.html'. We didn't quite understand what is the login page, how can one get into the login page (unless she directly type /login url)?